docs/guide/yaml/example_yaml/brooklyn-elk-catalog.bom

```diff
@@ -9,27 +9,28 @@ brooklyn.catalog:
     name: ELK Stack
     id: ELK-Stack
     description: |
-      Simple ELK stack deployment: it installs ES, Kibana and lostash as a child of Apache Tomcat 8
+      Simple ELK stack deployment: provisions Elasticsearch, Kibana and Logtash as a child of Apache Tomcat 8
     services:
       - type: elasticsearch
         id: es
-        name:  es
+        name:  Elasticsearch Cluster
         brooklyn.config:
           install.version: 1.4.4
       - type: kibana-standalone
         id: kibana
-        name: kibana
+        name: Kibana Server
         customize.latch: $brooklyn:component("es").attributeWhenReady("service.isUp")
         brooklyn.config:
           kibana.elasticsearch.ip: $brooklyn:component("es").attributeWhenReady("host.address.first")
           kibana.elasticsearch.port: $brooklyn:component("es").config("elasticsearch.http.port")
       - type: org.apache.brooklyn.entity.webapp.tomcat.Tomcat8Server
-        id: jboss
+        id: tomcat
+        war: https://tomcat.apache.org/tomcat-8.0-doc/appdev/sample/sample.war
```

Comment:
best to leave out the war as it creates a dependency of that URL. The example works without it (you can access the tomcat admin UI).